File: Java/0001.java
public int[] twoSum(int[] numbers, int target) {
HashMap<Integer, Integer> map = new HashMap<Integer, Integer>();
int[] result = new int[2];

for (int i = 0; i < numbers.length; i++) {
	if (map.containsKey(numbers[i])) {
		int index = map.get(numbers[i]);
		result[0] = index ;
		result[1] = i;
		break;
	} else {
		map.put(target - numbers[i], i);
	}
}
return result;


}

File: Python/0001.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dict = {}
        for i, v in enumerate(nums):
            if target - v in dict:
                return dict[target - v] , i 
            elif v not in dict:
                dict[v] = i
        return []
